src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -174,18 +173,17 @@ class RtpReceiverImpl @JvmOverloads constructor(
                     name = "SRTP path"
                     predicate = PacketPredicate { !RTCPUtils.isRtcp(it.buffer, it.offset, it.length) }
                     path = pipeline {
-                        node(PacketParser("RTP parser") { RtpPacket(it.buffer, it.offset, it.length) })
-                        node(payloadTypeFilter)
+                        node(RtpParser())
                         node(tccGenerator)
+                        node(audioLevelReader)
```

Comment:
i'd like to put a TODO here to something of the effect of:
```
TODO: temporarily putting the audioLevelReader node here such that we can determine whether or not a 
packet should be discarded before doing decrypt.  audioLevelReader has been moved here to avoid parsing 
the RTP header extensions twice (which is expensive).  In the future we will parse and cache the header 
extensions to make this lookup more efficient, at which time we can move audioLevelReader back to where it 
was (in the audio path) and add a new node here which can check for different discard conditions (e.g. 
checking the audio level for silence)
```

Reply:
haha, will do